website_partner_rating/controllers/main.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module uses OpenERP, Open Source Management Solution Framework.
+#    Copyright (C) 2014-Today BrowseInfo (<http://www.browseinfo.in>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+##############################################################################
+import werkzeug
+from openerp.addons.web import http
+from openerp.addons.web.http import request
+import openerp.http as http
+from openerp.http import request
+import openerp.addons.website_crm_partner_assign.controllers.main as website_partner_main
+import werkzeug.urls
+import werkzeug.wrappers
+
+
+class website_partner_rating_comments(website_partner_main.WebsiteCrmPartnerAssign):
+    """ This method is overloaded for to add messaege_rate and short_description
+    in product.template"""
+    @http.route(['/partners/partner/comment/<int:partner_id>'], type='http',
```

Comment:
if you are not changing any route param you must remove them all to rely on the default ones: `@http.route()`